.gitmodules

```diff
@@ -4,3 +4,7 @@
 [submodule "vendor/Catch2"]
 	path = vendor/Catch2
 	url = https://github.com/catchorg/Catch2
+[submodule "vendor/small_vector"]
+	path = vendor/small_vector
+	url = https://github.com/gharveymn/small_vector.git
+	branch = main
```

Comment:
I guess this was auto-generated, but what does `branch` do here exactly? The checked out revision is not stored in this file anyway. Just curious.

Reply:
I don't know, as you assumed this is generated and I just kept it as-is.